envoy/extensions/filters/http/header_rewrite/header_rewrite.pb.go

```diff
@@ -0,0 +1,148 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.30.0
+// 	protoc        v4.23.4
+// source: envoy/extensions/filters/http/header_rewrite/header_rewrite.proto
+
+package header_rewrite
+
+import (
+	_ "github.com/envoyproxy/protoc-gen-validate/validate"
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+type HeaderRewrite struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	Config string `protobuf:"bytes,1,opt,name=config,proto3" json:"config,omitempty"`
```

Comment:
Is the config a single line? 
Is this expected to change in a non-backward compatible way in the future?

Reply:
The config is a single string literal that could span multiple lines in the `yaml` file where it is specified. I don't foresee this changing, as the filter plugin gets all its configuration from one string value and parses it internally.

Reply:
My understanding is that this is for the current testing phase, and would likely change to a proper config for productionalization. Sounds okay for this phase